android-test/build.gradle

```diff
@@ -0,0 +1,31 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    repositories {
+        // Insert local test repo here
+        jcenter()
+        google()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.5.0-beta05'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.40"
+    }
+}
+
+allprojects {
+    repositories {
+        jcenter()
+        google()
+    }
+}
+
+ext {
+    kotlinVersion = "1.3.40"
```

Comment:
Don't like that all of these are separate from the other variables in the enclosing project